recipes/gperftools/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class GperftoolsConan(ConanFile):
+    name = "gperftools"
+    license = "BSD-3-Clause License"
+    homepage = "https://github.com/gperftools/gperftools"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "gperftools: originally Google Performance Tools"
+    topics = ("malloc", "profiler")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"fPIC": False, "shared": True}
+    autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
```

Comment:
Remove fPIC when shared + you need libtool to run autoreconf.
```suggestion
            del self.options.fPIC

    def configure(self):
        if self.options.shared:
            del self.options.fPIC

    def build_requirements(self):
        self.build_requires("libtool/2.4.6")

    def source(self):
```